src/_00_Sorting_Algorithms/_01_SwappingDigits.java

```diff
@@ -50,6 +58,16 @@ public static void sortIntArray(int[] arr) {
 	// *Hint* it helps to sort it first.
 	// *Double Hint* Use the method you already wrote in step 2 to sort it
 	public static int findMiddle(int[] arr) {
-		return 0;
+		int tempOldDigit = 0;
```

Comment:
This code looks like it's repeated from the method above.  Can you make it DRYer?